tsconfig.json

```diff
@@ -53,6 +53,7 @@
             "@share/*": ["share_extension/*"],
             "@store": ["app/store/index"],
             "@store/*": ["app/store/*"],
+            "@support/*": ["detox/e2e/support/*"],
```

Comment:
Do we need this here and not in the detox folder?

Reply:
Whenever I run npm run check from project root, it also goes through the detox folder. I added it there just so there are no errors in the check.